engine/src/main/java/org/hibernate/search/engine/search/highlight/dsl/CommonHighlightBoundaryScannerLocaleOptionStep.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.highlight.dsl;
+
+import java.util.Locale;
+
+/**
+ * TODO
+ */
+public interface CommonHighlightBoundaryScannerLocaleOptionStep<T extends CommonHighlightOptionStep<?>> {
+
+	default T defaultLocale() {
+		return locale( Locale.ROOT );
+	}
+
+	T locale(Locale locale);
```

Comment:
It's a schame that one *has* to specify the locale... it won't scale if we need to add more options related to the boundary scanner.